community-plugins.json

```diff
@@ -12430,5 +12430,14 @@
         "author": "Minh Vương",
         "description": "Auto rename pasted images added to the vault via hash algorithm SHA-512",
         "repo": "hardingadonis/hash-pasted-image"
-    }
+    },
+    {
+	"id": "text-composer-for-obsidian",
+	"name": "Text Composer",
+	"author": "Your Name",
+	"description": "A plugin to compile .md documents from linked documents.",
+	"repo": "filipaldi/text-composer-for-obsidian",
+	"version": "1.0.0",
+	"minAppVersion": "0.9.12"
```

Comment:
This doesn't belong here. You need to include this in your manifest.json file. Let me know if you have any questions!

```suggestion
```